packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -57,7 +57,7 @@ export interface IFluidDataStoreRuntime
 
 	readonly connected: boolean;
 
-	readonly logger: ITelemetryLogger;
+	readonly logger: ITelemetryLoggerExt;
```

Comment:
I think this one wants to be an `ITelemetryBaseLogger`. Ideally, a *-definitions package shouldn't depend on a package with functionality/implementations, so adding a dep on telemetry-utils here seems incorrect.

Adding to that, this logger is something that consumers of Fluid get access to, so following the logic that they shouldn't be exposed to `*Ext` types, `ITelemetryBaseLogger` seems more appropriate.

Reply:
Yep, should be `Base`

Reply:
Updated